pkg/build/types/types.go

```diff
@@ -69,6 +69,16 @@ type Architecture struct{ s string }
 
 func (a Architecture) String() string { return a.s }
 
+func (a *Architecture) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var buf string
+	if err := unmarshal(&buf); err != nil {
+		return nil
```

Comment:
should this return err?

Reply:
🤦🏼‍♀️

Reply:
Fixed in 59398a, ci failure appears due to an issue in https://github.com/chainguard-dev/actions/pull/35